samples/package.json

```diff
@@ -10,12 +10,19 @@
     "*.js"
   ],
   "scripts": {
-    "test": "mocha --timeout 600000 test/*.js"
+    "fix": "gts fix",
```

Comment:
So traditionally, we keep `gts` out of the samples dependency, and keep the `fix` and `lint` commands up at the top level package.  The linter comes with a lot of files like `.prettierignore`, and `.eslintignore` that need to be toted about.  I would like to keep things consistent here.  Can we remove it?

Reply:
Absolutely.